### PR TITLE
feat: Get User submitted repos

### LINKED ISSUE #353: Create logic to check if a user has submitted the repository (GET)
## What is this?

When submitting a repo to the app, we will need to check if the user has submitted it. This will be needed to identify the most popular users submitting repositories. 

## What is needed

A hook is needed to fetch https://api.opensauced.pizza/#/Submit%20service/findAllUserSubmitted

At the time of creating this issue, API hooks only exist on the alpha branch. Check here for a reference https://github.com/open-sauced/hot/tree/alpha/src/hooks

In this list, add an item that says `Submissions: count`

### PR DESCRIPTION
**Pull request by ikezuby2012**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
created a hook to fetch submitted repos. leveraging the repo's API

this PR fixes #353 
## Mobile & Desktop Screenshots/Recordings
WORK-IN-PROGRESS

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -276,3 +276,6 @@ build
 **/__snapshots__
 .env*
 !.env
+
+#local history folder
+/.history
```
Comment (bdougie) [inline .gitignore:281]: What in the IDE is happening here?

File: src/App.tsx [hunk 2]
```diff
@@ -26,6 +29,40 @@ console.log(
...
+      return map;
+    }
+
+    // console.log("You are on the server");
+
+    // 👉️ can't use localStorage
+
+    return (new Map);
+  };
+
```
Comment (bdougie) [inline src/App.tsx:65]: Might need to rebase beta. This might not be needed.

File: .gitignore (hunk 1 above)
Comment (ikezuby2012) [reply .gitignore:281]: Lol, its vs code local history folder i ignored.

**Review by bdougie — APPROVED** [review]
Would love to get @janie-lee-developer to also review this to see if this works with what she is working on.

**Review by 0-vortex — CHANGES_REQUESTED** [review]
The hook needs the authentication header if we want to merge this individually 🍕

**Comment by bdougie** [comment]
We are moving in a different direction on this. Sorry got the long delay, but definitely join us in the discord for updates.

https://discord.com/invite/U2peSNf23P

**Pull request closed without merging**